deploy/helm/charts/values.yaml

```
@@ -3,7 +3,7 @@
 # Declare variables to be passed into your templates.
 
 release:
-  version: "2.12.2"
+  version: "3..0.0"
```

It should be 3.0.0 right? Similarly in all other places
```suggestion
  version: "3.0.0"
```

---

Oops missed that. Thanks :+1: